src/main/java/com/dreammaster/gthandler/recipes/AssemblingLineRecipes.java

```diff
@@ -1,19 +1,19 @@
 package com.dreammaster.gthandler.recipes;
 
-import static gregtech.api.enums.Mods.GraviSuite;
-import static gregtech.api.util.GT_RecipeBuilder.HOURS;
-import static gregtech.api.util.GT_RecipeBuilder.MINUTES;
-import static gregtech.api.util.GT_RecipeBuilder.SECONDS;
-import static gregtech.api.util.GT_RecipeBuilder.WILDCARD;
+import static gregtech.api.enums.Mods.*;
```

Comment:
do not have wild imports.

Reply:
yea my IDE loves wildcard imports i guess